api.py

```diff
@@ -0,0 +1,32 @@
+import json
+from django.contrib.auth.decorators import user_passes_test
+from django.core import serializers
+from django.db import transaction, connection
+from django.http import HttpResponse
+from django.views.decorators.cache import never_cache
+from django.views.decorators.csrf import csrf_protect
+
+import commands
+
+@csrf_protect
+@never_cache
+@transaction.atomic
+@user_passes_test(lambda u: u.is_staff)
+def command(request):
+	data = json.loads(request.POST.get('data', '{}'))

```

Comment:
If we want to start doing PEP8 then we need to be using an editor that checks this kind of thing for us.

Which actually means 4-space indentation.